UK/LogonDetails.py

```diff
@@ -44,6 +57,58 @@
 CPDLC = input("Enter Hoppie's Code: ")
 os.system('cls')
 
+# VCCS PTT Key
+
+print("Push a key you want to use for VCCS. (Note: Keyboard only)")
+PTT = str(int(hex(keyboard.read_event().scan_code), 16) << 16)
```

Comment:
Missed this in my cursory look over earlier; what's the purpose of this? Is `int(hex(_), 16)` not identity?